pwndbg/dbg/lldb/pset.py

```diff
@@ -1,49 +1,40 @@
 from __future__ import annotations
 
-import argparse
 from typing import Any
 
 import pwndbg
 import pwndbg.color.message as message
 import pwndbg.commands
 import pwndbg.lib.config as cfg
 
-parser = argparse.ArgumentParser(description="Changes a Pwndbg setting.")
-parser.add_argument(
-    "name",
-    type=str,
-    default=None,
-    help="Name of the setting to be changed",
-)
-parser.add_argument(
-    "value",
-    type=str,
-    default=None,
-    help="Value to change the setting into",
-)
 
-
-@pwndbg.commands.ArgparsedCommand(parser)
-def pset(name, value):
+def pset(name: str, value: str) -> bool:
+    """
+    Parses and sets a Pwndbg configuration value.
+    """
     name = name.replace("-", "_")
     if name not in pwndbg.config.params:
         print(message.error(f"Unknown setting '{name}'"))
-        return
+        return False
 
     param = pwndbg.config.params[name]
     try:
         new_value = parse_value(param, value)
-    except InvalidParse:
-        print(message.error("Invalid value '{value}' for setting '{name}': {e.message}"))
-        return
+    except InvalidParse as e:
+        print(message.error(f"Invalid value '{value}' for setting '{name}': {e.message}"))
+        return False
 
     param.value = new_value
     for trigger in pwndbg.config.triggers[param.name]:
         trigger()
 
+    return True
+
 
 class InvalidParse(Exception):
-    pass
+    def __init__(self, message: str):
+        self.message = message
+        super().__init__()
```

Comment:
Shouldn't this whole init be:
```suggestion
        super().__init__(message)
```